source/guide_couchdb.rst

```diff
@@ -0,0 +1,148 @@
+.. highlight:: console
+
+.. author:: Lukas Wolfsteiner <lukas@wolfsteiner.media>
+
+.. tag:: self-hosting
+.. tag:: database
+
+.. sidebar:: About
+
+.. image:: _static/images/couchdb.svg
+      :align: center
+
+#######
+CouchDB
+#######
+
+.. tag_list::
+
+Apache CouchDB_ is an open-source document-oriented NoSQL database, implemented in Erlang.
+
+CouchDB uses multiple formats and protocols to store, transfer, and process its data, it uses JSON to store data, JavaScript as its query language using MapReduce, and HTTP for an API.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`the shell <basics-shell>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * http://www.apache.org/licenses/LICENSE-2.0
+
+Prerequisites
+=============
+
+We're using :manual:`CouchDB <database-couchdb>` in the stable version 3:
+
+.. code-block:: bash
```

Comment:
Use `console` instead of `bash` to fix the highlighting. Alternatively, just use `::` to default to `console`.
```suggestion
::
```
Same for all code blocks below.

Reply:
All of the other code blocks still have broken formatting: https://deploy-preview-959--uberlab.netlify.app/guide_couchdb.html#environment-setup